plugins/optimization-detective/class-od-url-metric.php

```diff
@@ -96,11 +96,13 @@ private function prepare_data( array $data ): array {
 			throw new OD_Data_Validation_Exception(
 				esc_html(
 					sprintf(
-						/* translators: 1: current aspect ratio, 2: minimum aspect ratio, 3: maximum aspect ratio */
-						__( 'Viewport aspect ratio (%1$s) is not in the accepted range of %2$s to %3$s.', 'optimization-detective' ),
+						/* translators: 1: current aspect ratio, 2: minimum aspect ratio, 3: maximum aspect ratio, 4: viewport width, 5: viewport height */
+						__( 'Viewport aspect ratio (%1$s) is not in the accepted range of %2$s to %3$s. Viewport dimensions: %4$sx%5$s', 'optimization-detective' ),
```

Comment:
Why is it important to include the dimensions in the message here?

Related: Maybe use a single placeholder in the string for `WIDTHxHEIGHT`? That portion doesn't need to be translatable individually.

Reply:
It's not important, but it explains how the aspect ratio was computed.

I've combined the width and height in 38f2e70e